guide/voice/audio-resources.md

```diff
@@ -0,0 +1,88 @@
+# Audio Resources
+
+Audio resources contain audio that can be played by an audio player to voice connections.
+
+## Cheat sheet
+
+### Creation
+
+There are many ways to create an audio resource. Below are some example scenarios:
+
+```ts
+import { createAudioResource, StreamType } from '@discordjs/voice';
+import { createReadStream } from 'fs';
+
+// Basic, default options are:
+// Input type is unknown, so will use FFmpeg to convert to Opus under-the-hood
+// Inline volume is opt-in to improve performance
+let resource = createAudioResource('file.mp3');
+
+// Will use FFmpeg with volume control enabled
+resource = createAudioResource('file.mp3', { inlineVolume: true });
+resource.volume.setVolume(0.5);
+
+// Will play .ogg or .webm Opus files without FFmpeg for better performance
+// Remember, inline volume is still disabled
+resource = createAudioResource(createReadStream('./file.ogg', {
+	inputType: StreamType.OggOpus
+}));
+
+// Will play with FFmpeg due to inline volume being enabled.
+resource = createAudioResource(createReadStream('./file.webm', {
+	inputType: StreamType.WebmOpus,
+	inlineVolume: true
+}));
+
+player.play(resource);
+```
+
+### Deletion
+
+The underlying streams of an audio resource are destroyed and flushed once an audio player is done playing their audio. Make sure to remove any references that you have created to the resource to prevent memory leaks.
+
+## Handling errors
+
+For most scenarios, you will create an audio resource for immediate use for an audio player. This means that the audio player will propagate errors from the resource for you, so you can attach `error` handlers to the player instead of the resource.
+
+However, it is still recommended to attach `error` handlers to the resource's `playStream`. In the event that you create your stream a little earlier than it is actually played, errors could still arise.
+
+:::warning
+Altering the state of an audio player from an audio resource error handler is **not** recommended. Your state changes may be reverted by the audio player's underlying error handler. State changes should only come from an audio player's error handler.
+:::
+
+A good solution for handling errors from these resources is to simply apply a `noop` handler to them. As long as you are listening for errors from your audio player, this is acceptable. This makes the assumption that you do not care about errors emitted from resources that aren't being consumed by an audio player.
+
+### Optimizations
```

Comment:
It seems to me that the optimizations header should be on its own level (i.e. `##`) instead of a child to "Handling errors"